inc/autoload/class-add-css.php

```diff
@@ -31,13 +31,14 @@ public function __construct() {
 
 		add_action( 'admin_head-themes.php', array( $this, 'enqueue_style' ) );
 		add_action( 'admin_head-plugins.php', array( $this, 'enqueue_style' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_style' ) );
```

Comment:
If we use this hook is the result that we lood the css in each admin screen. So you should add a check for admin screen to return early without load styles if the page is wrong or we use the hook only for specific pages.
What you htink about that?